core/field_multilineinput.js

```diff
@@ -402,16 +405,9 @@ FieldMultilineInput.prototype.onHtmlInputKeyDown_ = function(e) {
  */
 Css.register([
   /* eslint-disable indent */
-  '.blocklyHtmlTextAreaInput {',
-    'font-family: monospace;',
-    'resize: none;',
-    'overflow: hidden;',
-    'height: 100%;',
-    'text-align: left;',
-  '}',
-  '.blocklyHtmlTextAreaInputOverflowedY {',
-    'overflow-y: scroll;',
-  '}'
+  '.blocklyHtmlTextAreaInput {', 'font-family: monospace;', 'resize: none;',
+  'overflow: hidden;', 'height: 100%;', 'text-align: left;', '}',
+  '.blocklyHtmlTextAreaInputOverflowedY {', 'overflow-y: scroll;', '}'
```

Comment:
This is a lot less readable, is it possible to revert easily? If not then nbd.

Reply:
I forgot to check the css. Thanks for pointing it out I can fix it and I'll address in other PRs.